frame/support/procedural/src/pallet/parse/storage.rs

```diff
@@ -612,6 +618,16 @@ fn process_unnamed_generics(
 				false,
 			)
 		},
+		StorageKind::CountedNMap => {
+			let keygen = retrieve_arg(1)?;
```

Comment:
just flagging that merely having this word in a text file can cause some AV scanners to flag it so might want to rename this variable 😆

Reply:
I see your point, but not sure that renaming in one place would solve the problem, otherwise it is needed to rename in all places, not sure that it is a good idea in case of this PR